### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata descriptions, added missing column headers, and updated invalid alt text descriptions.

### CODE REVIEW

File: BingMaps/articles/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Bing Maps Articles | Microsoft Docs"
+description: "Provides an in index of Bing Maps articles with a table that provides a description for various articles relating to Bing Maps."
```
Comment (rflynn27) [inline BingMaps/articles/index.md:3]: remove "in"

File: BingMaps/getting-started/bing-maps-dev-center-help/geocoding-and-managing-data-sources/getting-data-source-information.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Getting Data Source Information | Microsoft Docs"
+description: "Describes process of getting data source information, outlines transaction accounting, and provides a reference for understanding Bing Maps transactions."
```
Comment (rflynn27) [inline BingMaps/getting-started/bing-maps-dev-center-help/geocoding-and-managing-data-sources/getting-data-source-information.md:3]: *the process of

**Comment by dereklh7** [comment]
@stevemunk @rbrundritt @jesselgit 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by dereklh7** [comment]
@stevemunk I have implemented requested changes.

#sign-off

**Comment by dereklh7** [comment]
@stevemunk @rbrundritt @jesselgit

I've resolved all merge conflicts for this PR. Please let me know if there are any other issues preventing approval and merge for this PR. Thanks!

**Pull request merged**